tispark-overview.md

```diff
@@ -206,9 +206,49 @@ tisparkDF.write.saveAsTable("hive_table") // save table to hive
 spark.sql("select * from hive_table a, tispark_table b where a.col1 = b.col1").show // join table across Hive and Tispark
 ```
 
+## Write DataFrame batches to TiDB using TiSpark
+
+Starting from v2.3, TiSpark natively supports writing DataFrame batches to TiDB clusters. This writing mode is implemented through two-phase commit protocol of TiKV.
+
+Compared to Spark + JDBC writes, TiSpark batch writes have the following features:
+
+|  Comparative aspects     | TiSpark batch writes | Spark + JDBC writes|
+| ------- | --------------- | --------------- |
+| Atomicity   | The data in the DataFrame is either all written successfully or all written unsuccessfully. | If the Spark task fails to exit during the writing process, part of the data is written successfully. |
+| Isolation   | During the writing process, other transactions are invisible to the data being written. | During the writing process, some successfully written data are visible to other transactions.  |
+| Error recovery | If batch writes fail, you only need to re-run the Spark. | Idempotence is achieved through business. For example, if batch writes fail, you need to clean up part of the successfully written data, and then re-run the Spark. You need to set `spark.task.maxFailures=1` to prevent data duplication caused by retry in the task. |
```

Comment:
```suggestion
| Error recovery | If the batch write fails, you only need to re-run Spark. | An application is required to achieve idempotence. For example, if the batch write fails, you need to clean up the part of the successfully written data and re-run Spark. You need to set `spark.task.maxFailures=1` to prevent data duplication caused by retry in the task. |
```